.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
our goal would be to get rid of TODO - usually it's added to old project where running `rubocop -A` would create too many changes - but here we are just starting so please try to fix as many cops as possible within this PR! Thanks 🙇 :)

Reply:
Rubocop cops fixes

Fixed safe auto-correction cops